clients/cobol-lsp-vscode-extension/package.json

```
@@ -90,7 +90,8 @@
                 "filenamePatterns": [
                     "**/*copybook*/**"
                 ],
-                "configuration": "./syntaxes/lang-config.json"
+                "configuration": "./syntaxes/lang-config.json",
+                "firstLine": "^([0-9 ]{6}[*].*|[0-9 ]{6}[ ].*\\b[Ii][Dd][Ee][Nn][Tt][Ii][Ff][Ii][Cc][Aa][Tt][Ii][Oo][Nn] [Dd][Ii][Vv][Ii][Ss][Ii][Oo][Nn]\\b.*)$"
```

are multiple spaces allowed?
```suggestion
                "firstLine": "^([0-9 ]{6}[*].*|[0-9 ]{6}[ ].*\\b[Ii][Dd][Ee][Nn][Tt][Ii][Ff][Ii][Cc][Aa][Tt][Ii][Oo][Nn] +[Dd][Ii][Vv][Ii][Ss][Ii][Oo][Nn]\\b.*)$"
```